apps/sasquatch/src/main/java/com/microsoft/appcenter/sasquatch/activities/MSALoginActivity.java

```diff
@@ -193,17 +193,8 @@ public void onReceivedError(WebView view, WebResourceRequest request, WebResourc
 
     private void clearCookies() {
         CookieManager cookieManager = CookieManager.getInstance();
-        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP_MR1) {
```

Comment:
lollipop_mr1 = 22, while we're bumping our minsdk to 21. Can we really remove this clause?
If the previous version was correct the 'else' clause should be valid for android API 21.